README.md

```
@@ -223,124 +259,23 @@ My portfolio and other open source projects → [santifer.io](https://santifer.i
-## Inicio rapido
-
-```bash
-# 1. Clonar
-git clone https://github.com/santifer/career-ops.git
-cd career-ops && npm install
+## Contributors
 
-# 2. Verificar setup
-npm run doctor                     # Valida todos los prerequisitos
+<a href="https://github.com/santifer/career-ops/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=santifer/career-ops" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)